godot-macros/Cargo.toml

```diff
@@ -20,3 +23,6 @@ proc-macro2 = "1.0.63"
 quote = "1.0.29"
 
 venial = "0.5"
+
+[build-dependencies]
+godot-bindings = { path = "../godot-bindings" } # emit_godot_version_cfg
```

Comment:
This is needed, but unfortunately forces the `godot-macro` crate compilation to begin only after `godot-bindings` has finished, possibly increasing compile times.

 We should probably (in a separate PR, out of scope here) extract the version detection and `#[cfg]` generation to its own crate, also in light of #281.